.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow builds the docker container and publishes to dockerhub with appropriate tag
+# It has two triggers, 
+#   1. daily i.e. runs everyday at specific time.
+#   2. Anytime a new branch is created under releases
+
+name: Publish Feathr Docker image to DockerHub
+
+on:
+  schedule:
+    # Runs daily at 10 PM UTC, would generate nightly tag
+    - cron: '00 22 * * *'
+  
+  push:
+    # For every push against the releases/** branch, usually would happen at release time, Tag example - releases/v0.7.0
+    branches:
+      - 'releases/**'
```

Comment:
Looks like current both triggers are configured on release branch. I think nightly image should be generated from main branch?

Reply:
There is a schedule trigger which runs at 10 pm UTC and it is triggered on main not on release branch, if you don't specify a branch it defaults to main. Currently this builds update two tags, nightly and main.

![image](https://user-images.githubusercontent.com/8405222/184577905-e36a308b-1757-40f4-901c-67667f841d74.png)